### PR TITLE
Latin6 RGB use IS31FL3731

### PR DESCRIPTION
**Pull request by 18438880**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: latin6RGB/rules.mk [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+RGBLIGHT_ENABLE = no           # Enable keyboard RGB underglow
+
+BLUETOOTH_ENABLE = no          # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no              # Audio output on port B7
+FAUXCLICKY_ENABLE = no         # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no            # Enable support for HD44780 based LCDs
+
+NO_USB_STARTUP_CHECK = no      # Disable initialization only when usb is plugged in
+
+LAYOUTS = numpad_2x3
```
Comment (zvecr) [inline latin6RGB/rules.mk:36]: Not a community layout
```suggestion

```

File: latin6RGB/rules.mk [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (zvecr) [inline latin6RGB/rules.mk:11]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]
All files and folders need to be lower case, per https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=naming-your-keyboardproject and https://docs.qmk.fm/#/pr_checklist?id=general-prs

**Review by zvecr — CHANGES_REQUESTED** [review]
Along with the lowercase thing, you have also added to the wrong location. files should be within the `keyboards` folder.

**Comment by 18438880** [comment]
> Along with the lowercase thing, you have also added to the wrong location. files should be within the `keyboards` folder.

SORRY sir,This is my mistake,I will correct it,thanks a lot.

**Pull request closed without merging**